rfcs/0146-meta-categories.md

```diff
@@ -0,0 +1,298 @@
+---
+feature: Decouple filesystem from categorization
+start-date: 2023-04-23
+author: Anderson Torres
+co-authors:
+shepherd-team: @7c6f434c @natsukium @fgaz @infinisil
```

Comment:
```suggestion
shepherd-team: @7c6f434c @natsukium @fgaz
```

I'd like to retract my involvement as an RFC shepherd. After [discussing this in the previous Nixpkgs Architecture Team meeting](https://discourse.nixos.org/t/2023-10-31-nixpkgs-architecture-team-meeting-45/34846), I've concluded that it's not something I'm very interested in. My passion is mostly in solving the most prominent existing problems in Nixpkgs. This RFC however is more about introducing something new, which might also become a prominent problem if not done correctly, but for now is not.

Furthermore, the more things I'm simultaneously doing, the less focus I have, the less efficient I am, making everything just take longer in the end (also me being a bit over-perfectionistic doesn't help).

I can review this RFC again when/if it reaches FCP, but I intend to be open about the approach and not insist on anything I mentioned previously.

Reply:
Thanks for the work you have done as a shepherd, and also thanks for all the arguments you have provided, regardless of their status!

Reply:
I feel a bit sad, however I understand your motivations.
Nonetheless, many thanks for the input. I will think about the debtags.

Reply:
@AndersonTorres I think you should accept the suggested change here…

Reply:
@AndersonTorres I think you need to accept this suggestion before the FCP ends, so that the final shepherd list in the repository is the list of those who have formally signed off on starting the FCP

Reply:
wow, I thought it was already done!